qa/automation/CareerPortal.Tests.Scenarios/Burlaka.feature

```diff
@@ -0,0 +1,119 @@
+# Dear Reviewer, my version of VS2022 does not support text enlightening, so I want to provide you with a link to my Online Specflow files.
```

Comment:
Sorry if my comments souds like rude. I am not good in English.

Reply:
Thank you for the comments! They have contributed to my understanding of mistakes that have to be corrected. :)